common/src/main/java/com/sequenceiq/cloudbreak/quartz/configuration/QuartzJobInitializer.java

```diff
@@ -20,6 +21,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(QuartzJobInitializer.class);
 
+    private static final int QUARTZ_INIT_LOCK_NUMBER = 'Q' + 'u' + 'a' + 'r' + 't' + 'z';
```

Comment:
Wouldn't it be better to set an exact number and avoid accidental conflicts? Let's say I need to write a locking functionality for some other reason and I construct a lock number like this but for some other string I get the same integer, now I will use the same lock (which I shouldn't be).